src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml

```diff
@@ -400,11 +400,21 @@
                     
                     <te:SectionControl.Content>
                         <Grid>
+                            <Grid.Resources>
+                                <!-- We have to do contortions to get to the PullRequestDetailViewModel which contains the OpenFile and DiffFile
+                                     commands: the control which hosts the context menu must have the PullRequestDetailViewModel as its tag,
+                                     which is then set as the tag on the ContextMenu which is finally reacable from the MenuItem. -->
+                                <ContextMenu x:Key="FileContextMenu" Tag="{Binding PlacementTarget.Tag, RelativeSource={RelativeSource Mode=Self}}">
```

Comment:
In views that need to trigger commands, I usually put a command on the view that just redirects back to whoever is the actual handler, so the xaml doesn't get overly complicated, like the `OpenPR` command in the `PullRequestListView`. Would that help here?